README.md

```diff
@@ -12,15 +12,20 @@ The collection includes a variety of Ansible content to help automate the manage
 
 This collection has been tested against following Ansible versions: **>=2.14.0**.
 
+For collections that support Ansible 2.9, please ensure you update your `network_os` to use the
+fully qualified collection name (for example, `cisco.ios.ios`).
+Plugins and modules within a collection may be tested with only specific Ansible versions.
+A collection may contain metadata that identifies these versions.
```

Comment:
I guess this can be probably removed.

Reply:
Ensuring the use of fully qualified collection names aligns with one of the rules outlined in ansible-lint. I've noticed this requirement in a few other collections too. Could we rephrase this guideline? What are your thoughts?

Reply:
Is there any collections that support Ansible 2.9?

Reply:
I do not think so. I can rephrase the line while excluding the support for Ansible 2.9.